src/Components/Articles.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+import ChangePage from "./SwitchPage";
+
+export default function Articles() {
+  const [articleList, setArticleList] = useState([]);
+  const [page, setPage] = useState(1)
+
+  useEffect(() => {
+    axios
+      .get(`https://northcoders-news-03ck.onrender.com/api/articles?p=${page}`)
```

Comment:
you should extract this out into an api.js file so you can just call the function in your useEffect.